filebeat/input/udp/input.go

```diff
@@ -268,7 +268,7 @@ func procNetUDP(addr []string) (rx, drops int64, err error) {
 	}
 	for _, l := range lines[1:] {
 		f := bytes.Fields(l)
-		if contains(f[1], addr) {
+		if len(f) > 12 && contains(f[1], addr) {
```

Comment:
`12` is a little "magical",  how about at least a link to kernel docs for why there should be this many fields?
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/net/ipv4/udp.c?h=v4.14.312#n2764

also fmt.Errorf above should probably use the `path` variable instead of "/proc/net/udp" now.

Reply:
The reason I checked for 12 is because it is highest field indexed (e.g. `f[12]`) by this section of code.

> fmt.Errorf above should probably use the path

Yeah, that thought crossed my mind. I didn't do it because it was parameterized only for testing purposes. But if you had the thought too then I will change it.